trio/_dtls.py

```diff
@@ -1146,15 +1195,15 @@ def __init__(self, socket, *, incoming_packets_buffer=10):
         # as a peer provides a valid cookie, we can immediately tear down the
         # old connection.
         # {remote address: DTLSChannel}
-        self._streams = weakref.WeakValueDictionary()
-        self._listening_context = None
-        self._listening_key = None
-        self._incoming_connections_q = _Queue(float("inf"))
+        self._streams: weakref.WeakValueDictionary = weakref.WeakValueDictionary()
```

Comment:
Shouldn't there be generic arguments here?

Reply:
ye, I hadn't enabled `disallow_any_generics`